da/newda/newda.go

```diff
@@ -0,0 +1,119 @@
+package newda
```

Comment:
Was this added as a `newda` package for ease of implementation?

Is this something we can revert to us `da`? `newda` seems like double naming

Reply:
yes, this will be renamed to just da in the next PR.